.circleci/config.yml

```diff
@@ -61,17 +57,18 @@ jobs:
       # Install dependencies.
       # For unclear reasons, 'jazzy' fails to install on CircleCI if 'atomos' is not installed first.
       - run: sudo gem install atomos jazzy
-      - run: brew install awscli cmake
+      - run: brew install cmake jfrog-cli-go
       # Build the framework in release mode and package it into pod.zip
       - run: make ios-framework-universal RELEASE=1
       - run: make ios-docs
-      - run: cd build/ios-framework-universal/Release/ && zip -r ${CIRCLE_ARTIFACTS}/pod.zip TangramMap.framework
-      # Add the readme file.
-      - run: cd platforms/ios/framework/ && zip ${CIRCLE_ARTIFACTS}/pod.zip README.md
-      # Add the license file.
-      - run: zip ${CIRCLE_ARTIFACTS}/pod.zip LICENSE
-      # Upload the Cocoapods archive to S3.
-      - run: aws s3 cp ${CIRCLE_ARTIFACTS}/pod.zip s3://ios.mapzen.com/tangram-releases/tangram-release-${CIRCLE_TAG}.zip
+      - run: zip -r pod.zip build/ios-framework-universal/Release/TangramMap.framework
+      # Add the readme and license files.
+      - run: zip pod.zip platforms/ios/framework/README.md LICENSE
+      # Upload and publish the Cocoapods archive to Bintray.
+      - run: jfrog bt config --user=${BINTRAY_USER} --key=${BINTRAY_API_KEY} --licenses=MIT
```

Comment:
I assume you will be adding `BINTRAY_USER` and `BINTRAY_API_KEY` to circleci environment?

Reply:
Similar for travis.

Reply:
Ok Confirmed these environment variables are added to the 2 ci environments.
Will also make sense to clean the environment variables on ci to reflect the ones not being used now. Example MAPZEN_API_KEY or S3 credentials.

Reply:
Yeah I will clean up the CI environment variables after this is merged.